src/pages/calculator/settings/index.tsx

```diff
@@ -1,8 +1,48 @@
+import React, { useCallback } from "react"
+
 import CalculatorLayout from "@/components/layouts/calculator"
 import { ComponentWithLayout } from "@/components/layouts/types"
+import { useAppDispatch, useAppSelector } from "@/store/hooks"
+import {
+  selectTravelerGender,
+  setTravelerGender,
+  TravelerGender,
+} from "@/store/settings/settingsSlice"
+import { Switch } from "@headlessui/react"
 
 export const SettingsPage: React.FC & ComponentWithLayout = () => {
-  return <div className="flex justify-center w-full max-w-5xl">Settings</div>
+  const dispatch = useAppDispatch()
+  // const [twin, setTwin] = useState<TravelerGender>(useAppSelector(selectTravelerGender))
+  const twin: TravelerGender = useAppSelector(selectTravelerGender)
+
+  const setTwin = useCallback(
+    (gender) => {
+      dispatch(setTravelerGender(gender))
+    },
+    [dispatch],
+  )
+
+  return (
+    <Switch.Group as="div" className="flex space-x-4">
+      <Switch.Label>Select twin. Currently: {twin}</Switch.Label>
+      <Switch
+        as="button"
+        checked={twin === "female"}
+        onChange={() => setTwin(twin === "male" ? "female" : "male")}
+        className={`${
```

Comment:
Use the `clsx` library here to do conditional className string concatenation. You can look at other components on how it's done.

typical structure:
```
className={
    clsx(
        "default styles first",
        condition ? condition true style : condition false style
    )
}
```